student.rb

```diff
@@ -1,9 +1,13 @@
 require_relative 'person'
+require_relative 'classroom'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission: parent_permission)
     @classroom = classroom
+    @classroom.add_student(self) if classroom
   end
 
```

Comment:
Please add a `method` that serves as the 'belongs-to' side (a student belongs to a classroom)

![image](https://github.com/ankitt26/School-Library/assets/78251927/a07d0108-cd94-4165-9606-69e7c943c540)

As you have created the `has-many` side (a classroom has many students). Please create the `belongs-to` side (a student belongs to a classroom) also (Both ways relationships)

```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```



Link to the article with examples: [Here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)

Reply:
Thank you for bringing this to my attention. I am updating the code.